wagtail/users/forms.py

```diff
@@ -394,7 +395,11 @@ class Meta:
 
 
 def _get_language_choices():
-    return sorted(BLANK_CHOICE_DASH + get_available_admin_languages(),
+    language_choices = [
+        (lang_code, get_language_info(lang_code)['name_local'])
+        for lang_code, lang_name in get_available_admin_languages()
```

Comment:
👍  ... if we're not using lang_name here any more, then we can hopefully remove [the localisations on that list](https://github.com/wagtail/wagtail/blob/main/wagtail/admin/localization.py#L11) that were probably creating mostly-useless work for translators :-)

Reply:
done in https://github.com/gasman/wagtail/commit/1e9d32915b8c95167256d17ba1bdc95b2295d941